app/assets/javascripts/application/submission.js

```diff
@@ -14,7 +14,6 @@ var mumuki = mumuki || {};
     },
     success: function (data, submitButton) {
       this.submissionsResultsArea.html(data.html);
-      mumuki.showKidsResult(data);
```

Comment:
Why?

Reply:
The success callback doesn't get executed when the submissions is sent by the runner and the runner is already calling the showKidsResults method.

Reply:
OK, seems legit. And I like that we are removing explicit calls to kids more here. 

However, please see my comments in https://github.com/mumuki/mumuki-gobstones-runner/pull/78#pullrequestreview-109360204